language-support/hs/bindings/BUILD.bazel

```diff
@@ -36,13 +36,11 @@ da_haskell_library(
 daml_compile(
     name = "for-tests",
     srcs = glob(["test/daml/for-tests/*.daml"]),
-    main_src = "test/daml/for-tests/Main.daml",
 )
 
 daml_compile(
     name = "for-upload",
     srcs = glob(["test/daml/for-upload/ExtraModule.daml"]),
```

Comment:
Isn't this dangerous? If the file doesn't exist, then `glob` means we'll just get the empty list potentially obscuring an error?

Reply:
good point. I'll raise an explicit error for `[]`

Reply:
Bazel itself now errors on empty globs unless you explicitly allow them with [`allow_empty`](https://docs.bazel.build/versions/master/be/functions.html#glob).

Reply:
A DALF doesn’t need to have any modules. I think you just want to omit the glob and use the list directly here.

Reply:
> Bazel itself now errors on empty globs unless you explicitly allow them with [`allow_empty`](https://docs.bazel.build/versions/master/be/functions.html#glob).

unfortunately the `allow_empty` defaults to True, so it's not very useful as a mistake catcher

I fixed the case where a glob wasn't necessary.

Reply:
> unfortunately the `allow_empty` defaults to True, so it's not very useful as a mistake catcher

Indeed, they've been pushing the switch to default to `allow_empty = False` to later and later times. Last I checked it should already have been switched in 2.0, but it looks like now it's scheduled for Bazel 4.0, see https://github.com/bazelbuild/bazel/issues/8195.